entity-api-spec.yaml

```diff
@@ -2095,6 +2394,10 @@ paths:
                 type: array
                 items:
                   type: object
+              examples:
+                collectionarray:
+                  value: 
+                    - collection: 'Donor object 1'
```

Comment:
The result of `GET /entities/{id}/donors` is a json array of organ objects, and not related to any `collection` key.